test/feature/finding_controllers.feature

```diff
@@ -34,3 +34,21 @@ Feature: Finding controllers
     When making a GET request to "/users"
     Then the response code should be 200
     And the response body should be "user list"
+
+
+  Scenario: an application with a javascript file in controllers not exporting a controller
+    Given a file "controllers/my_controller_spec.coffee" with the content
+      """
+      # This file doesn't export a controller
+      """
+    And an exprestive app using defaults
+    Then the app doesn't error
+
+
+  Scenario: an application with a non-javascript file in controllers
+    Given a file "controllers/readme.md" with the content
+      """
+      Readme about controllers
+      """
+    And an exprestive app using defaults
+    Then the app doesn't error
```

Comment:
You have 3 criteria for controllers (@isJsFile() and @exportsName() and @exportsConstructor()), but only 2 scenarios here.